src/components/Layout/UserDropdown/MiniQuotaDisplay/index.tsx

```diff
@@ -0,0 +1,86 @@
+import Infinity from '@app/assets/infinity.svg';
+import { SmallLoadingSpinner } from '@app/components/Common/LoadingSpinner';
+import ProgressCircle from '@app/components/Common/ProgressCircle';
+import type { QuotaResponse } from '@server/interfaces/api/userInterfaces';
+import { Suspense } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import useSWR from 'swr';
+
+const messages = defineMessages({
+  movierequests: 'Movie Requests',
+  seriesrequests: 'Series Requests',
+});
+
+type MiniQuotaDisplayProps = {
+  userId: number;
+};
+
+const MiniQuotaDisplay = ({ userId }: MiniQuotaDisplayProps) => {
+  const intl = useIntl();
+  const { data } = useSWR<QuotaResponse>(`/api/v1/user/${userId}/quota`, {
+    suspense: true,
+  });
+
+  return (
+    <Suspense fallback={<SmallLoadingSpinner />}>
+      {(data?.movie.limit ?? 0) !== 0 && (data?.tv.limit ?? 0) !== 0 && (
```

Comment:
On profile pages, we display that both movie & series requests are unlimited. Do we not want to do the same here?

Reply:
No reason to show it if both are unlimited. It's just dead space at that point.

Reply:
I was thinking more that if we end up making these shortcuts to view one's own movie/series requests, that it would be helpful to have these elements exist regardless.

Or, what do you think about adding a `Requests` entry along with `Profile` and `Settings` in the dropdown that links to `/profile/requests` (currently missing but added as a fix in #2190)?

Reply:
Good catch. And because the type is a number so it's better to compare with numbers even though 0 is still falsey

Reply:
> I was thinking more that if we end up making these shortcuts to view one's own movie/series requests, that it would be helpful to have these elements exist regardless.
> 
> 
> 
> Or, what do you think about adding a `Requests` entry along with `Profile` and `Settings` in the dropdown that links to `/profile/requests` (currently missing but added as a fix in #2190)?

I'm okay with adding requests!